docs/cases/lyon.md

```diff
@@ -126,6 +147,7 @@ is used, choose the updated HTS in the config:
 config:
   # ...
   hts: edgt_lyon
+  edgt_lyon_source: adisp  # adisp/cerema 
```

Comment:
Can you add a comment somewhere in the text above to make sure it is clear that one needs to make a choice here?